### DISCUSSION TITLE
OpenAI 推出超神 ChatGPT 注册攻略来了

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#97] [Announcements]
# OpenAI 推出超神 ChatGPT 注册攻略来了

OpenAI 推出超神 ChatGPT，非常火爆。但是呢，因为不可抗力原因，大部分人无法体验到。这里我分享一下注册的攻略。 准备 首先能能访问 Google（前置条件，不能明确说，懂得都懂） 你得有一个国外手机号，GV 号肯定不行。 如果你没有国外手机号，推荐sms-activate.org    注册短信平台并充值 先行注册sms-activate.org 注册好之后进行对应的充值   接码费用

https://youngjuning.js.org/d744d1d3d1e8/

**Comment by ytppp** [comment]
学到了

**Reply by youngjunning** [reply]
很高兴能帮到你

**Comment by weicun581** [comment]
hello

**Reply by youngjunning** [reply]
hello bro
